ui/app/AppLayouts/Onboarding/panels/AccountMenuItemPanel.qml

```diff
@@ -58,6 +58,8 @@ MenuItem {
             image: root.image
             colorId: root.colorId
             colorHash: root.colorHash
+            imageHeight: 40
```

Comment:
@faust4exe please add to `Contacts.qml`, new `QtObject` property, named `onboarding` and let's keep within it those values referring to login/onboarding flow like, userImageWidth, userImageHeight, userDisplayNameFontSize and so...

Reply:
Done - and I guess you meant `Constants.qml` :)